website/content/en/v1.0/upgrading/v1-migration.md

```diff
@@ -669,6 +669,7 @@ Revisit step 9 of the [upgrade procedure]({{< ref "#upgrading" >}}) and ensure t
   * NodePools now have [status conditions]({{< relref "../concepts/nodepools/#statusconditions" >}}) that indicate if they are ready. If not, then they will not be considered during scheduling.
   * NodeClasses now have [status conditions]({{< relref "../concepts/nodeclasses/#statusconditions" >}}) that indicate if they are ready. If they are not ready, NodePools that reference them through their `nodeClassRef` will not be considered during scheduling.
   * Karpenter will no longer set `associatePublicIPAddress` to false in private subnets by default. Users with IAM policies / SCPs that require this field to be set explicitly should configure this through their `EC2NodeClass` ([ref]({{<ref "../concepts/nodeclasses/#specassociatepublicipaddress">}})).
+  * Due to Node expiration, you may observe an increased number of pods in the "Scheduling" state.
```

Comment:
I'm wondering if we should actually include a more detailed section in [Before You Start](https://karpenter.sh/v1.0/upgrading/v1-migration/#before-you-start) on forceful expiration. By going into detail there, I think we can describe both what the change is, and what the implications are. I think this warrants that additional level of detail - from this line alone the implications aren't super clear.

Reply:
I assume that this is meant to capture that due to forceful expiration, Nodes which were previously expired but not disrupted (due to budgets, PDBs, `do-not-disrupt` pods, etc) will be disrupted simultaneously. At that point, the scheduler will begin provisioning nodes for the pods on those disrupted nodes, which may not be able to drain due to factors which were blocking disruption in the first place.